modules/navier_stokes/test/tests/ins/lid_driven/tests

```diff
@@ -70,4 +70,42 @@
     issues = '#000'
     requirement = 'We shall be able to solve the INS equations using the classical Chorin splitting algorithm'
   [../]
+
+  [./lid_driven_action]
+    type = 'Exodiff'
+    input = 'lid_driven_action.i'
+    exodiff = 'lid_driven_out.e'
+    custom_cmp = 'lid_driven.cmp'
+    prereq = 'lid_driven'
+    issues = '#15159'
+    requirement = 'We shall be able to reproduce the results from the input with MOOSE syntax only using the Modules/IncompressibleNavierStokes syntax'
+  [../]
+  [./lid_driven_stabilized_action]
+    type = 'Exodiff'
+    input = 'lid_driven_stabilized_action.i'
+    exodiff = 'lid_driven_stabilized_out.e'
+    custom_cmp = 'lid_driven_stabilized.cmp'
+    prereq = 'lid_driven_stabilized'
+    issues = '#15159'
+    requirement = 'We shall be able to reproduce the results from the input with MOOSE syntax only using the Modules/IncompressibleNavierStokes syntax'
+  [../]
+  [./ad_lid_driven_action]
+    type = 'Exodiff'
+    input = 'ad_lid_driven_action.i'
+    exodiff = 'lid_driven_out.e'
+    custom_cmp = 'lid_driven.cmp'
+    method = 'OPT'
+    prereq = 'ad_lid_driven'
+    issues = '#15159'
+    requirement = 'We shall be able to reproduce the results from the input with MOOSE syntax only using the Modules/IncompressibleNavierStokes syntax'
+  [../]
+  [./ad_lid_driven_stabilized_action]
+    type = 'Exodiff'
+    input = 'ad_lid_driven_stabilized_action.i'
+    exodiff = 'lid_driven_stabilized_out.e'
+    custom_cmp = 'lid_driven_stabilized.cmp'
+    prereq = 'ad_lid_driven_stabilized'
+    issues = '#15159'
+    requirement = 'We shall be able to reproduce the results from the input with MOOSE syntax only using the Modules/IncompressibleNavierStokes syntax'
+  [../]
```

Comment:
This is great, thank you. Gives me the confidence that the results are the same!

The requirement phrasing is a little unclear to me. Moreover we are trying to make the requirement language be as separate as possible from the nature of the implementation. E.g. using an implementation name like `Modules/IncompressibleNavierStokes` is not ideal. Also, it looks like you've just used the same requirement four times :smile: 

I'd like to see something like: "The system shall be able to reproduce hand-coded Jacobian, unstabilized Navier-Stokes results using a customized and condensed action syntax." And then replace the first bit of that sentence depending on which requirement you're writing. Also, make sure you're adding periods to the end of requirements.

I'm only being strict about this because @permcody has hammered me on it many times.

Reply:
Are you sure we need periods at the end? Looks like all other tests do not have period, ;-)

Reply:
Thanks for adding the periods to all!